PyXiaomiGateway/__init__.py

```diff
@@ -143,7 +143,7 @@ def _listen_to_msg(self):
                 cmd = data['cmd']
                 if cmd == 'heartbeat' and data['model'] == 'gateway':
                     gateway.token = data['token']
-                elif cmd == 'report' or cmd == 'heartbeat':
+                elif cmd == 'report':  # or cmd == 'heartbeat': Commented Out to avoid false positive statuses
```

Comment:
Please remove the comment and commented out part. Your commit message must explain your change entirely.